src/endpoints/create.rs

```diff
@@ -54,6 +54,7 @@ pub async fn create(
         extension: String::from(""),
         private: false,
         editable: false,
+        non_removable: false,
```

Comment:
I don't fully understand the use of this attribute here. Right now it seems like it is only ever assigned to but never queried?

Isn't the idea of this PR to disable the removal of pastas entirely, rather than on a pasta-by-pasta basis? In that case, I'd say this field should be removed.

Reply:
Will update and PR this weekend @albocc

Reply:
I saw your update few minutes ago. I'd say it looks good to me now 👍

Reply:
Yes, but it's including the field in db.

![image](https://user-images.githubusercontent.com/24442168/209216822-5475e755-4d2e-4e0c-bb7d-a023c0a85acf.png)

idk, where it's coming from.

Reply:
Did you create those entries with an older code revision? I just checked out your fork and for me the `non_removable` field was not there, so I assume it was created by an older version?

Reply:
No, I:

- deleted the old built binary
- deleted db directories
- rebuilt

Reply:
That's the only explanation for me if the string `non_removable` does not show up anywhere in the checked in code? 😮

Reply:
ummm, It's fine now.. Tested in a sandbox environment 
![Screenshot_20221223_021037](https://user-images.githubusercontent.com/24442168/209222805-71e121f5-71d6-4e82-b469-0df178d1a1b2.png)

 🫡